build/bwm-ng/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2022 OmniOS Community Edition (OmniOSce) Association.
+
+. ../../lib/build.sh
+
+PROG=bwm-ng
+VER=0.6.3
+PKG=ooce/network/$PROG
+SUMMARY="CLI network and disk io monitor"
+DESC="small and simple console-based live network and disk io bandwidth monitor"
+
+set_arch 64
+set_mirror "$GITHUB/vgropp/$PROG/archive"
+set_checksum "none"
+
+init
+download_source v$VER $PROG $VER
```

Comment:
I've put this onto the mirror now.

```suggestion

init
download_source $PROG v$VER
```